homeassistant/components/starline/binary_sensor.py

```diff
@@ -18,7 +19,8 @@
     BinarySensorEntityDescription(
         key="hbrake",
         translation_key="hand_brake",
-        device_class=BinarySensorDeviceClass.POWER,
+        entity_category=EntityCategory.DIAGNOSTIC,
```

Comment:
What does the hand brake sensor measure?

Reply:
Handbrake on or off

Reply:
Isn't that a non diagnostic measurement then?

Reply:
Why not?

Reply:
I'd categorize the hand break on/off state the same as the door closed/open state and other car measurements that pertain to the driver using the car.

Reply:
As you wish